addons/procurement/product_product_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="product_template_view_form_procurement_button" model="ir.ui.view">
+            <field name="name">product.template_procurement_stat_button</field>
```

Comment:
Generally we use dots, not underscores (product.template.procurement. ...). Same for below.